### DISCUSSION TITLE
transfert lent

### DISCUSSION THREAD

**Discussion by MOUTMOUT69** [localsend/localsend#557] [L-Français]
je rencontre des problèmes de débit lors de transfert entre 2 pc sous windows  via une connexion en wifi 6 je ne dépasse pas les 4Mb/s ce qui rend les transfert extrêmement long il y a t il des choses a paramétrer ??
en vous remerciant

**Comment by thomas1615** [comment]
Moi, entre mon PC et mon tel, le transfert est très rapide. Il n'y a pas photo, entre LocalTransfert et par exemple SendAnywhere, en plus open source et pas de pub.
